.github/workflows/reusable_guidelines_enforcer.yml

```diff
@@ -31,21 +37,22 @@ jobs:
     secrets:
       git_token: ${{ secrets.git_token }}
 
-  call_check_is_rust:
-    name: Check if app is based on Rust
-    uses: ./.github/workflows/_check_is_rust.yaml
+  call_get_app_metadata:
```

Comment:
The workflow files are intended for external devs, we should make them more explicit
Add a comment here explaining this helper job